src/healthkit.cs

```diff
@@ -171,6 +202,54 @@ public interface HKCategorySample {
 		HKCategorySample FromType (HKCategoryType type, nint value, NSDate startDate, NSDate endDate, [NullAllowed] HKDevice device, [NullAllowed] NSDictionary<NSString,NSObject> metadata);
 	}
 
+	[Watch (3,0), iOS (10,0)]
+	[BaseType (typeof(HKSample))]
+	[DisableDefaultCtor] // NSInvalidArgumentException Reason: The -init method is not available on HKDocumentSample
+	interface HKDocumentSample
+	{
+		[Export ("documentType", ArgumentSemantic.Strong)]
+		HKDocumentType DocumentType { get; }
+	}
+
+	[NoWatch, iOS (10,0)]
+	[BaseType (typeof(HKDocumentSample))]
+	[DisableDefaultCtor] // NSInvalidArgumentException Reason: The -init method is not available on HKCDADocumentSample
+	interface HKCDADocumentSample
```

Comment:
I did call it `HKCdaDocumentSample` but got a null handle with the `CheckHandle` test. Then I looked at the generated code which did `Class.GetHandle ("HKCdaDocumentSample")`, which I believe is incorrect. Do I have to do anything special here?